src/main/java/com/reservation/event/domain/SessionOccurrenceCancelled.java

```diff
@@ -0,0 +1,6 @@
+package com.reservation.event.domain;
+
+import java.time.Instant;
+
+public record SessionOccurrenceCancelled(SessionOccurrenceId id, Instant createdAt) implements SessionOccurrenceEvent {
```

Comment:
Zgodnie z zasadą [program to interface](https://www.baeldung.com/cs/program-to-interface) warstwa aplikacyjna będzie pracowała z typem `SessionOccurrenceEvent` więc nie musi znać żadnej implementacji tego interfejsu, i dlatego interface możemy ustawić na publiczny, natomiast wszystkie jego implementacje pozostają niewidoczne dla tej warstwy, dzięki temu osiągamy luźniejsze powiązanie.

Warstwa aplikacyjna zna tylko interfejs domeny, a nie szczegóły.
```suggestion
record SessionOccurrenceCancelled(SessionOccurrenceId id, Instant createdAt) implements SessionOccurrenceEvent {
```